### PR TITLE
[WB-3694] - remove symlink for windows

### PR DESCRIPTION
**Pull request by KyleGoyette**
Link to JIRA issue
https://wandb.atlassian.net/browse/WB-3694

### CODE REVIEW

File: wandb/sdk/wandb_init.py [hunk 1]
```diff
@@ -286,6 +298,19 @@ def _log_setup(self, settings):
...
+                settings.log_user,
+                os.path.basename(settings.log_symlink_user),
+                delete=True,
+            )
+            self._safe_copy(
+                os.path.dirname(settings.log_symlink_internal),
+                settings.log_internal,
+                os.path.basename(settings.log_symlink_internal),
+                delete=True,
+            )
```
Comment (raubitsj) [inline wandb/sdk/wandb_init.py:313]: These arent needed, they are just convenience symlinks.  copy would be bad as they would copy a file that doesnt have any log data in them

**Review by raubitsj — CHANGES_REQUESTED** [review]

File: wandb/sdk/wandb_init.py (hunk 1 above)
Comment (KyleGoyette) [reply wandb/sdk/wandb_init.py]: I see now, removed

**Review by raubitsj — CHANGES_REQUESTED** [review]
@KyleGoyette 
This seems fine, but I don't understand how to test this change.

Do you have a tensorboard test script that I can run on windows to reproduce the failure and reproduce the fix?  I run tensorboard training from wandb/examples and wandb/wandb-examples and they never seem to hit the symlink issue (i dont think i am using an elevated user)

Why were the wandb_test changes necessary? is it because circle runs our windows tests as admin.  Does this fix when unittests are run as a user?

File: wandb/sdk/internal/tb_watcher.py [hunk 2]
```diff
@@ -50,12 +50,17 @@ def _link_and_save_file(
...
-        os.symlink(abs_path, wandb_path)
+    if settings.symlink:
+        print("DOING SYMLINKS")
+        if os.path.islink(wandb_path) and abs_path != os.readlink(wandb_path):
+            print("REMOVING AND ADDING")
+            os.remove(wandb_path)
+            os.symlink(abs_path, wandb_path)
+        elif not os.path.exists(wandb_path):
+            print("JUST SYMLINKING")
+            os.symlink(abs_path, wandb_path)
```
Comment (vanpelt) [inline wandb/sdk/internal/tb_watcher.py:63]: Printzzz

**Review by vanpelt — APPROVED** [review]

**Comment by kptkin** [comment]
@KyleGoyette suggested that we can close it for now

**Pull request closed without merging**